client/components/platform-checkout/save-user/style.scss

```diff
@@ -70,6 +71,55 @@
 	}
 }
 
+#phone-number {
+	height: 42px;
+	background-color: #f2f2f2;
```

Comment:
I'm not sure if this is even something worth moaning about here, but on a dark background, perhaps with dark mode, this section looks a bit weird. Here's what it looks like to me: I think this is still the standard Storefront, with perhaps a little bit of theme tinkering. 

![Dark illegible inputs](https://cdn-std.droplr.net/files/acc_1104206/Ulosln)
_Text illegible with dark background styling_

Here it is if we inherit the background colour on the parent `.save-details-form`.

![Legible text fields](https://cdn-std.droplr.net/files/acc_1104206/bNZZVq)
_Legible inputs_

I'm not sure if this is worth worrying about in this PR, but I noticed it, so felt like mentioning it.

Reply:
Addressed this one by removing the bg and using an existing class which is originally applied to the `place order` section.